websockets/conversation.go

```diff
@@ -154,11 +178,28 @@ func (c *Conversation) handleEditUpdate(msg protocol.Message, sender *Client) er
 	for client := range c.clients {
 		if client != sender {
 			client.caret = client.caret.ShiftCaret(sender.caret, *update.Delta)
+			newCheckpoint.syncsLeft[client.userID] = true
+		}
+		newCheckpoint.activeUsers[client.userID] = client.caret
+	}
+
+	c.checkpoint[*update.Version] = newCheckpoint
+
+	if len(newCheckpoint.syncsLeft) == 0 {
```

Comment:
This only gets triggered in the scenario where only one user has the conversation open right?

Reply:
Right